src/fmu/dataio/scripts/copy_preprocessed.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+
+"""Copy preprocessed data to an FMU case while updating the metadata.
+
+This script is intended to be run through an ERT HOOK PRESIM workflow.
+
+"""
+
+from __future__ import annotations
+
+import argparse
+import logging
+import warnings
+from pathlib import Path
+from typing import TYPE_CHECKING, Final
+
+from fmu.dataio import ExportPreprocessedData
+
+try:
+    from ert.shared.plugins.plugin_manager import hook_implementation
+except ModuleNotFoundError:
+    from ert_shared.plugins.plugin_manager import hook_implementation
+
+try:
+    from ert.config import ErtScript
+except ImportError:
+    from res.job_queue import ErtScript
+
+if TYPE_CHECKING:
+    from ert.shared.plugins.workflow_config import WorkflowConfigs
+
+logger: Final = logging.getLogger(__name__)
+logger.setLevel(logging.CRITICAL)
```

Comment:
Any reasons for setting this?

Reply:
No not really.. just leftover from copying the `WF_CREATE_CASE_METADATA` as template for this new job. 
I will remove it 🙂